lesson-7-rails-lab-reports/app/controllers/users_controller.rb

```diff
@@ -0,0 +1,51 @@
+class UsersController < ApplicationController
+  before_action :set_user, only: %i[show edit update destroy]
+
+  def index
+    @previous_page = ["Lab Reports", lab_reports_path]
```

Comment:
```suggestion
    @previous_page = [ "Lab Reports", lab_reports_path ]
```
Сам рубокоп жалуется)
![image](https://github.com/user-attachments/assets/15aa7de1-02f0-4cc8-b975-67416f1ed349)